blockdiag/Dockerfile

```diff
@@ -16,6 +16,9 @@ RUN apk --update add --virtual build-dependencies build-base python3-dev jpeg-de
 
 COPY src .
 
+RUN addgroup -g 1000 kroki && adduser -D -G kroki -u 1000 kroki
```

Comment:
Could you please move this statement just after `FROM`?
The reason is that some packages could potentially add a new user/group and take id 1000.